tests/data/granule.umm_json

```diff
@@ -132,5 +132,24 @@
             "Date": "2024-03-02T22:12:36.000Z",
             "Type": "Update"
         }
-    ]
+    ],
+    "Platforms": [
+            "Sentinel-1A",
```

Comment:
Each entry in `Platforms` will be a dictionary describing a few things about the satellite. Here's what I&A used for the OPERA RTC data set, which I think we should copy:

```suggestion
        {
          'ShortName': 'Sentinel-1A',
          'Instruments': [
            {
              'ShortName': 'C-SAR',
              'Characteristics': [
                {
                  'Name': 'LookDirection',
                  'Value': 'RIGHT',
                },
              ],
              'OperationalModes': [
                'IW',
              ],
            },
          ],
        }
```

The only difference between S1A and S1B will be the `ShortName`.

If the input granules are all from S1A, `Platforms` only needs a single item from S1A. If the input granules are all from S1B, `Platforms` only needs a single item for S1B. If the input granules are from a mix of S1A and S1B, `Platforms` should have one item for S1A and a second item for S1B.